.github/workflows/docker.yml

```diff
@@ -0,0 +1,54 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches: ['dev']
+  pull_request:
+    branches: ['dev']
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  display-image-name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Display IMAGE_NAME
+        run: echo "IMAGE_NAME is ${{ env.IMAGE_NAME }}"
```

Comment:
Do we have tag w/ image name? or we are just overwriting one image?

Reply:
Tag is defined in the section`Build and push Docker image`. The convention after discussion with devops is to use {branch-short_commit_hash}